args.js

```diff
@@ -12,7 +12,7 @@ module.exports = function parseArgs (args) {
       'populate--': true
     },
     number: ['port', 'inspect-port', 'body-limit', 'plugin-timeout'],
-    string: ['log-level', 'address', 'socket', 'prefix', 'ignore-watch', 'logging-module', 'debug-host', 'lang', 'require'],
+    string: ['log-level', 'address', 'socket', 'prefix', 'ignore-watch', 'logging-module', 'debug-host', 'lang', 'require', 'standardlint'],
```

Comment:
At least I see you declare it as string option.

Reply:
Good point. That's because I started to code thinking of it to be a string indeed, then I thought "if someone really tries to put more options here, it will be annoying AF to decide and maintain each non-standard option" and so I changed my mind.

but, again, I'm just waiting for some agreed final design choice from all reviewers, it should be an easy code change anyway.